lib/capifony_symfony1.rb

```diff
@@ -149,7 +149,6 @@ def generate_sql_command(cmd_type, config)
     symfony.orm.build_classes             # 2. (Re)build the model
   end
   symfony.cc                              # 3. Clear cache
-  symfony.plugin.publish_assets           # 4. Publish plugin assets
```

Comment:
Tested with symfony 1.4. No surprise: without this line, plugin assets are not available via symlink in the web dir.